dxr/app.py

```diff
@@ -162,6 +164,27 @@ def _tree_tuples(trees, tree, query_text, is_case_sensitive):
              description)
             for t, description in trees.iteritems()]
 
+@dxr_blueprint.route('/<tree>/raw/<path:path>')
+def send_raw(tree, path):
```

Comment:
Let's just call this `raw`. The rest of the routes are pretty much just named after their URL segments.